app/assets/stylesheets/modules.css.scss

```diff
@@ -23,6 +23,20 @@
   }
 }
 
+.resource-block {
+  background:transparent image_url("noun_116372.png") 90% 50% no-repeat;
+  background-size: 100px;
+
+  a {
+    &.text { background-color:#F5F5EB; background-color:rgba(245, 245, 235, 0.95); }
+    &.video { background-color:#EBF5EB; background-color:rgba(235, 245, 235, 0.95); }
+    &.tutorial { background-color:#EBEBF5; background-color:rgba(235, 235, 245, 0.95); }
```

Comment:
Color literals like `#EBEBF5` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Property `background-color` already defined on line 33<br>Color `#EBEBF5` should be written as `#ebebf5`<br>Colon after property should be followed by one space